testing/index.test.js

```diff
@@ -100,4 +104,46 @@ it('should return 5 reviews', async () => {
           }
       ]
   });
-});
\ No newline at end of file
+});
+
+it('reviews 5 should be fast', async() => {
+  await pactum.spec()
+    .get('http://localhost:3000/reviews')
```

Comment:
Is there a way to test the data being received as well?
Just to make the tests more robusts and verify that you are receiving the expected data.